.github/ISSUE_TEMPLATE/0-scanner_run_bug.yml

```diff
@@ -77,7 +77,7 @@ body:
     description: |
       What do you get from the command "sf plugins"?
     placeholder: |
-      Example: @salesforce/sfdx-scanner 4.1.0 (latest-beta)
+      Example: @salesforce/sfdx-scanner 4.3.0 (latest)
```

Comment:
While I was in here, I figured I update our examples in our issue templates to not use a beta release as an example version.